.github/workflows/publish.yml

```diff
@@ -0,0 +1,18 @@
+name: Publish Package to npmjs
+on:
+  release:
+    types: [published]
+jobs:
+  npm-publish:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      # Setup .npmrc file to pub.nvmrclish to npm
+      - uses: actions/setup-node@v4
+        with:
+          node-version-file: '.nvmrc'
+          registry-url: 'https://registry.npmjs.org'
+      - run: npm ci
+      - run: npm publish --provenance --access public
```

Comment:
IS it possible that nove 16 doesn't support `--provenance`?  It says so according to [docs](https://docs.npmjs.com/generating-provenance-statements#prerequisites)

> Install the latest version of the npm CLI (ensure you are on 9.5.0+ as older versions don't support npm provenance). For more information, see "[Try the latest stable version of npm](https://docs.npmjs.com/try-the-latest-stable-version-of-npm)."

and according to [this](https://nodejs.org/en/about/previous-releases), npm has a version `8.x`

Reply:
Changed to node 18 and builds correctly.